tests/suites/examples/README.md

```diff
@@ -0,0 +1,20 @@
+# Example Test Suite
+
+The following is a very quick overview of what each file does and why.
+
+## task.sh
+
+`task.sh` file is the entry point to the suite. You can bootstrap a juju
+controller for use inside the suite. Each controller is namespaced, so running
+multiple controllers should be fine.
+
+Registration of tests are also done in this file.
+
+## example.sh
+
+`example.sh` describes a test in the test suite. Generally the grouping of tests
+inside of a file should be of the same ilk.
+
+## other.sh
+
```

Comment:
Would be great if we could just add a sample tree layout as kind of tl'dr:
Something like a general strucutre.
```
- test
-- task.sh -> entrypoint, creates controller
-- example.sh -> test itself, each test can be used with ensure to create a model
-- something -> ?
```

And use the tree for explaining.
What do you think?

Reply:
Do we want to add document some common functions the lib provided?
E.g.
- ensure
- bootstrap
- check_contains